extensions/bean/deployment/src/main/java/org/apache/camel/quarkus/component/bean/deployment/BeanProcessor.java

```diff
@@ -68,4 +69,19 @@ void registerReflectiveClasses(CombinedIndexBuildItem index, BuildProducer<Refle
         }
     }
 
+    @BuildStep
+    void registerBeanHandlersForReflection(BuildProducer<ReflectiveClassBuildItem> reflectiveClass,
+            CombinedIndexBuildItem combinedIndex) {
+        IndexView index = combinedIndex.getIndex();
+        DotName handlerAnnotation = DotName.createSimple(Handler.class.getName());
```

Comment:
This could be better to move to static var like `LANGUAGE_ANNOTATION`

Reply:
Yeah sure noted.